tfexec/errors.go

```diff
@@ -11,7 +11,7 @@ import (
 var (
 	// The "Required variable not set:" case is for 0.11
 	missingVarErrRegexp  = regexp.MustCompile(`Error: No value for required variable|Error: Required variable not set:`)
-	missingVarNameRegexp = regexp.MustCompile(`The root module input variable "(.+)" is not set, and has no default|Error: Required variable not set: (.+)`)
+	missingVarNameRegexp = regexp.MustCompile(`The root module input variable\s"(.+)"\sis\snot\sset,\sand\shas\sno\sdefault|Error: Required variable not set: (.+)`)
```

Comment:
Unless I'm mistaken, variables can't contain spaces, so would it make sense to instead just change `(.+)` to `([^ ]+)`? i.e. any character except for space? Would that satisfy the test and your needs?

Reply:
The issue is that any of the spaces after "The root module input variable" may be a newline. Here is an example of what I mean. You could have terraform output that looks like this:

> The root module input variable "a" is not set, and has no default

or this:

> The root module input variable "short_var" is not set, and has
> no default

or this:

> The root module input variable
> "super_long_variable_name_that_is_wrapped_to_its_own_line"
> is not set, and has no default

or it could look like this:

> The root module input variable "medium_name_var" is not set,
> and has no default

The line will wrap depending on the length of the variable name and what Terraform thinks the width of the terminal is. That can happen in multiple places in that error message (but always after the "The root module input variable" part of the string, which is why I'm not using \s before then).

Reply:
Got it, that makes sense - thank you for all the examples.